client/src/components/Login/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { authenticate } from '../../api/auth'
+
+type FormData = {
+  email: string
+  password: string
+}
+export default function App() {
+  const [responseError, setresponseError] = useState(null)
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<FormData>()
+
+  const onSubmit = handleSubmit(data => {
+    authenticate(data).then(
+      response => {
+        setresponseError(null)
+        //navigate to the main page
```

Comment:
good way to handle it as a placeholder, it would be better to change it to `// TODO: navigate to the main page`